### PR TITLE
Enable condor_chirp for workflows

### PR DESCRIPTION
**Pull request by duncan-brown**
This adds a Condor class ad to enable ``condor_chirp`` as described in the instructions at http://research.cs.wisc.edu/htcondor/manual/current/condor_chirp.html

### CODE REVIEW

File: pycbc/workflow/pegasus_files/osg-site-template.xml [hunk 1]
```diff
@@ -13,5 +13,6 @@
     <profile namespace="condor" key="should_transfer_files">YES</profile>
     <profile namespace="condor" key="when_to_transfer_output">ON_EXIT_OR_EVICT</profile>
     <profile namespace="condor" key="+OpenScienceGrid">True</profile>
+    <profile namespace="condor" key="+WantIOProxy">&quot;True&quot;</profile>
```
Comment (bbockelm) [inline pycbc/workflow/pegasus_files/osg-site-template.xml]: Same comment as above.

File: pycbc/workflow/pegasus_files/local-site-template.xml [hunk 2]
```diff
@@ -16,3 +16,4 @@
     <profile namespace="condor" key="use_x509userproxy">True</profile>
     <profile namespace="condor" key="+DESIRED_Sites">&quot;nogrid&quot;</profile>
     <profile namespace="condor" key="+IS_GLIDEIN">&quot;False&quot;</profile>
+    <profile namespace="condor" key="+WantIOProxy">&quot;True&quot;</profile>
```
Comment (bbockelm) [inline pycbc/workflow/pegasus_files/local-site-template.xml]: This isn't actually required to do `set_job_attr_delayed`.

Comment (duncan-brown) [inline pycbc/workflow/pegasus_files/local-site-template.xml]: Should we give ourselves the ability to do both?

Comment (bbockelm) [inline pycbc/workflow/pegasus_files/local-site-template.xml]: Nope, it's just going to kill the performance.  You don't need any changes to the job template at all -- just invoke `condor_chirp`.  See how CMS does it here:

https://github.com/cms-sw/cmssw/blob/master/FWCore/Services/plugins/CondorStatusUpdater.cc#L410

**Comment by duncan-brown** [comment]
@bbockelm thanks, will close this then.

**Pull request closed without merging**